src/Classes/Controllers/DeleteApplicantController.php

```diff
@@ -10,20 +10,54 @@ class DeleteApplicantController
 {
     private $applicantModel;
 
+    /**
+     * DeleteApplicantsController constructor saves an applicantModel
+     * onto this object
+     *
+     * @param ApplicantModel $applicantModel
+     */
     public function __construct(ApplicantModel $applicantModel)
     {
         $this->applicantModel = $applicantModel;
     }
 
+    /**
+     * Checks if user is logged in, validates the http request data and calls
+     * the delete method on applicantModel
+     *
+     * @param Request $request
+     * @param Response $response
+     * @param array $args
+     *
+     * @return \Slim\Http\Response - Returns status 200/500 with message in Json
+     */
     public function __invoke(Request $request, Response $response, array $args)
     {
-        $requestData = $request->getParsedBody();
-        if (isset($requestData['id'])) {
-            if ($this->applicantModel->deleteApplicant($requestData['id'])) {
-                return $response->withStatus(200);
+        if ($_SESSION['loggedIn'] === true) {
+            $data = ['success' => false, 'msg' => 'Applicant not found.'];
```

Comment:
Missing empty data array, even if unused its the standard format used elsewhere in the application